docker-entrypoint.sh

```diff
@@ -75,96 +132,47 @@ fi
 set +e
 export WP_CLI_CACHE_DIR="/tmp/.wp-cli/cache/"
 
-
 for i in {1..5}
-do
-  echo "Checking if DB is ready"
-  wp db check --allow-root
-  if [ $? -eq 0 ]; then
-    echo "Database is ready"
-    break
-  else
-    if [ $i -eq 5 ]; then
-      echo "Database is not ready after 5 attempts. Exiting"
-      exit 1
-    fi
-    echo "Database is not ready, waiting 10 seconds"
-    sleep 10
-  fi
-done
-
-wp core --allow-root is-installed
-INSTALLED=$?
-set -e
-if [ $INSTALLED -eq 0 ]; then
-    echo "Wordpress `wp core --allow-root version` is already installed, skipping installation"
-    # Print out the current version of Wordpress
-
-    runuser -u www-data -- mv /var/www/html/wp-content /var/www/html/wp-content_backup && echo "Successfully backed up wp-content" || echo "Failed to back up wp-content"
-    runuser -u www-data -- ln -s /mnt/data/wp-content /var/www/html && echo "Created a symlink to wp-content" || echo "Failed to create symlink to wp-content"
-
-    echo "Starting Apache"
-    exec "$@"
-else
-    echo "Wordpress is not installed, going to install"
-    if [ -z "$WORDPRESS_ADMIN_PASSWORD" ]; then
-      runuser -u www-data -- wp core install --skip-email --title="$WORDPRESS_TITLE" \
-       --url="$WORDPRESS_URL" --admin_user="$WORDPRESS_ADMIN_USER" \
-       --admin_email="$WORDPRESS_ADMIN_EMAIL" --path=/var/www/html/
-    else
-      runuser -u www-data -- wp core install --skip-email --title="$WORDPRESS_TITLE" --url="$WORDPRESS_URL" \
-      --admin_user="$WORDPRESS_ADMIN_USER" --admin_email="$WORDPRESS_ADMIN_EMAIL" \
-      --admin_password="$WORDPRESS_ADMIN_PASSWORD" --path=/var/www/html/
+ do
+   echo "Checking if DB is ready"
+   wp db check --allow-root
+   if [ $? -eq 0 ]; then
+     echo "Database is ready"
+     break
+   else
+     if [ $i -eq 5 ]; then
+       echo "Database is not ready after 5 attempts. Exiting"
+       exit 1
+     fi
+     echo "Database is not ready, waiting 10 seconds"
+     sleep 10
+   fi
+ done
+
+export WP_READY="/mnt/data/wp-ready"
+export WP_INIT_LOCK="/mnt/data/wp-init.lock"
+
+if [ ! -f "$WP_REAY" ]; then
```

Comment:
Typo?

Reply:
Yes. The typo was fixed. Image can work with this typo, but was starting and stopping temporary apache in all cases, not only if WP_READY file is absent.